shims.bzl

```diff
@@ -236,7 +239,11 @@ def cpp_binary(
         modules = None,
         **kwargs):
     _unused = (dlopen_enabled, compiler_specific_flags, os_linker_flags, allocator, modules)  # @unused
+    if os_deps:
+        deps += _select_os_deps(_fix_dict_deps(os_deps))
     prelude.cxx_binary(
+        name = name, # FIXME: The prelude cxx_binary() should allow this unnamed
+        srcs = srcs,
         deps = _fix_deps(deps + external_deps_to_targets(external_deps)),
         visibility = visibility,
         **kwargs
```

Comment:
`kwargs` here captures `name` and `srcs`. Why are you pulling them out if you're not modifying them?

Reply:
`name` is getting pulled out because it is required (see (docs)[https://buck2.build/docs/prelude/rules/cxx/cxx_binary/]). The failure is more direct and explicit if it is declared directly on line 229, but that requires the direct pass here.

`srcs` does not require the same, but I felt it made this more readable. Removed.